fr/index.html

```diff
@@ -17,22 +17,22 @@
     <h1>Ne demande pas la permission de demander, demande juste.</h1>
     <p>
       De temps en temps, dans des channels que je fréquente,
-      je vois quelqu'un apparaître et demander,
+      quelqu'un arrive et dit :
     </p>
     <blockquote>
       <span class="name">Foobar123 :</span>
       <p class="message">
-        Aucun expert Java dans le coin ?
+        Y a-t-il un expert Java dans le coin ?
```

Comment:
"Any Java experts around" was translated with the sentence that made me do the PR "Aucun expert Java dans le coin?" 

This isn't correct grammatically because you're supposed to start such questions with the verb in the interrogative form.

I proposed the translation "Y a-t-il un expert Java dans le coin ?", which, even if it contains a weird combination of letters and hyphens simply translates to "Is there [any] Java expert around".

In English you can omit the verb starting the sentence and making it implicit, thus giving the phrase "[is there] Any Java expert around?"

Furthermore, "Aucun" in the original translation translates to "None" "No", in the sense of a "lack of [something]. Implicitely a french speaker reading it could misunderstand and think the speaker is surprised that there are no Java experts around and asks about it rhetorically.

I believe my translation makes the text less ambiguous and more natural